src/PharoThings-Devices-HD44780.package/RpiBoard3B.extension/instance/configurePotLCD..st

```diff
@@ -8,8 +8,7 @@ configurePotLCD: aDevice
 	---------------------------------------------------------------
 	"
    aDevice
-		setPin: (self pinWithId: 27) at: #modePin; "RS 1 character or 0 lcd commands" 
-		setPin: (self pinWithId: 22) at: #clockPin; "EN clock enable"
-		setPin: (#(18 23 24 25) collect: [:id | self pinWithId: id]) at: #dataPins; "D7 D6 D5 D4 [D3 D2 D1 D0] data bit 4bit or [8bit]"
-		setPin: (self pinWithId: 12) at: #backlight; "BL back light PWM"
-		busMode: '#4bit'. "4bit or 8bit LCD bus mode"
\ No newline at end of file
+		modePin: (self pinWithId: 27); "RS 1 character or 0 lcd commands" 
+		clockPin: (self pinWithId: 22);"EN clock enable"
+		dataPins: (#(18 23 24 25) collect: [:id | self pinWithId: id]); "D7 D6 D5 D4 [D3 D2 D1 D0] data bit 4bit or [8bit]"
+		backlightPin: (self pinWithId: 12).  "BL back light PWM"
```

Comment:
And last point. I think it would be better to have double dispatch here back to the device:
```Smalltalk
RpiBoard3B>>configureComplexDevice: aDevice
    aDevice configureForRpiBoard3B

PotHD44780Controller configureForRpiBoard3B: aBoard
        modePin := board pinWithId: 27. 
        clockPin := board pinWith: 22.
        dataPins := #(18 23 24 25) collect: [:id | board pinWithId: id].
        backlightPin := board pinWithId: 12.
```
This way it would be subclass responsibility of boards to implement #configureComplexDevice and each device will implement set of methods representing supported boards.
Also it will avoid board pollution with different device names and duplication of those names (one used in class name itself and one in the name of configuration method). 
For example now you have very common name #configurePotLCD. But imagine other LCDs which requires other configuration. In common case each device will require new method in boards.

So with double dispatch all logic will be inside device itself. And boards will know nothing about them. Device developers will need to support known boards with set of methods